### PR TITLE
hw1 done

### PR DESCRIPTION
**Pull request by b14ckster**

### CODE REVIEW

**Comment by AleksandraMalygina** [comment]
It looks like 'EmptyTextsSuite' is running several times with the same parameters (both Test Cases)

File: hw1.xml [hunk 1]
```diff
@@ -0,0 +1,617 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<con:soapui-project id="72ebc347-87e1-42bc-a573-65ecba82b485" activeEnvironment="Default" name="lab1.yandex.speller" soapui-version="5.6.0" abortOnError="false" runType="SEQUENTIAL" resourceRoot="" xmlns:con="http://eviware.com/soapui/config"><con:settings><con:setting id="com.eviware.soapui.impl.wsdl.actions.iface.tools.soapui.TestRunnerAction@values-local"><![CDATA[<xml-fragment xmlns:con="http://eviware.com/soapui/config">
```
Comment (AleksandraMalygina) [inline hw1.xml]: work is needed  - see details in comments.

**Review by AleksandraMalygina — CHANGES_REQUESTED** [review]
look in the comments

File: hw1.xml (hunk 1 above)
Comment (b14ckster) [reply hw1.xml]: done

**Review by AleksandraMalygina — APPROVED** [review]
Good gob!

**Pull request merged**